packages/mds-transactions-service/tests/service.spec.ts

```diff
@@ -105,22 +107,64 @@ describe('Transaction Service Tests', () => {
     }
   })
 
+  function uuidv4() {
+    return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, c => {
+      const r = (Math.random() * 16) | 0
+      const v = c === 'x' ? r : (r & 0x3) | 0x8
+      return v.toString(16)
+    })
+  }
+
+  it('Bulk Post 20 Transactions', async () => {
+    const payloads: TransactionDomainModel[] = []
+    let timestamp = Date.now() - 20 * 1000
+    for (let i = 0; i < 20; i++) {
+      payloads.push({
+        transaction_id: uuidv4(),
+        provider_id,
+        device_id,
+        timestamp,
+        amount: 100, // "I'd buy THAT for a dollar!"
+        fee_type: 'base_fee',
+        receipt
+      })
+      timestamp += 1000
+    }
+    const transactions = await TransactionServiceClient.createTransactions(payloads)
+    expect(transactions[0].device_id).toEqual(device_id)
```

Comment:
Could use destructuring here if you like:
```suggestion
    const [transaction] = await TransactionServiceClient.createTransactions(payloads)
    expect(transaction.device_id).toEqual(device_id)
```